src/main/java/com/hivemq/extension/sdk/api/auth/parameter/EnhancedAuthOutput.java

```diff
@@ -127,10 +130,12 @@ public interface EnhancedAuthOutput extends AsyncOutput<EnhancedAuthOutput> {
     /**
      * Successfully authenticates the client.
      * <p>
-     * During authentication a CONNACK packet with reason code {@link com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS
+     * During authentication a CONNACK packet with reason code
+     * {@link com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS
      * SUCCESS} and no authentication data is sent to the client.
```

Comment:
why is this in new line?

Reply:
`com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS` can't be wrapped and exceeds the 120 limit, so it moves the whole link into a new line. Makes sense to me.

Reply:
I guess the formatter is again too stupid to reformat the whole paragraph in a more senseful way, e.g.:
```
     * During authentication a CONNACK packet with reason code {@link
     * com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS SUCCESS} and no authentication data is
     * sent to the client.
```